### PR TITLE
add CLI flag to assume yes to confirmation prompts

### PR DESCRIPTION
**Pull request by mafrosis**
..

### CODE REVIEW

File: saltcloud/utils/parsers.py [hunk 1]
```diff
@@ -204,6 +204,13 @@ def _mixin_setup(self):
             action='store_true',
             help='Update salt-bootstrap to the latest develop version on GitHub'
         )
+        group.add_option(
+            '-y', '--assume-yes',
+            default=False,
+            dest='assume_yes',
```
Comment (techhat) [inline saltcloud/utils/parsers.py]: Check out #266, and take note of the absence of dest.

Comment (mafrosis) [inline saltcloud/utils/parsers.py]: I was imitating the "--assume-yes" option from aptitude. The dash means it won't directly work as a variable name - so I used dest, as above on the "--no-deploy" option.

Comment (s0undt3ch) [inline saltcloud/utils/parsers.py]: Long `OptionParser` options have their `dest` set automatic to something like:

``` python
'--long-option'.lstrip('--').replace('-', '_')
```

If not explicitly set.

PS: Above is untested code, but you get the idea.

Comment (mafrosis) [inline saltcloud/utils/parsers.py]: Thanks for the pointer Pedro - I didn't know that.

**Comment by mafrosis** [comment]
Pull req updated to remove unnecessary config param

**Comment by techhat** [comment]
Excellent.

File: saltcloud/utils/parsers.py (hunk 1 above)
Comment (s0undt3ch) [inline saltcloud/utils/parsers.py]: If I can help :smiley:

**Pull request merged**